cmake/helpers/CheckDependentLibraries.cmake

```
@@ -443,6 +443,9 @@ gdal_check_package(IDB "enable ogr_IDB driver" CAN_DISABLE)
 gdal_check_package(rdb "enable RIEGL RDB library" CONFIG CAN_DISABLE)
 include(CheckDependentLibrariesTileDB)
 
+gdal_check_package(ExprTk "Enable C++ Mathematical Expression Tooklit Library (ExprTk) for VRT expressions" DISABLED_BY_DEFAULT)
+gdal_check_package(muparser "Enable muparser library for VRT expressions" CAN_DISABLE)
```

to hint packagers (although I suspect they won't pay much attention to that, so we should make some noise in the release notes)
```suggestion
gdal_check_package(muparser "Enable muparser library for VRT expressions" RECOMMENDED CAN_DISABLE)
```